3. Installation/4. Manual Installation/Ubuntu/Snaps/README.md

```diff
@@ -67,13 +67,59 @@ sudo systemctl restart snap.rocketchat-server.rocketchat-mongo
 
 The snaps policy is to restart on failure.
 
-### I need to backup the data from my snap how do I do this?
+### How do I backup my snap data?
 
-All data is located in `/var/snap/rocketchat-server/common`
+#### 1. Make a quick and easy backup via:
+```
+sudo snap run rocketchat-server.backupdb
+```
+
+#### 2. You will then see a bunch of output text followed by:
+```
+A backup of your data can be found at /var/snap/rocketchat-server/<version>/backup.tgz
+```
+
+#### 3. Copy `backup.tgz` to a different system for safekeeping.
+
+### How do I restore backup data to my snap?
+
+#### 1. Assuming you have your `backup.tgz` file (see above), simply extract it to a desired location (e.g., ~/backup_data)
+```
+cd ~
+mkdir backup_data
+tar zxvf /path/to/your/backup.tgz
+```
+
+This will create the following directory tree where you can find the extracted backup data:
+`~/backup_data/var/snap/rocketchat-server/<version>/dump/parties`
+
+#### 2. Confirm your database name
+The snap database name should be `parties`, but just to be safe:
+```
+sudo /snap/rocketchat-server/current/bin/mongo
```

Comment:
I do not. Mine a barebones Debian Jessie VM with testing repos enabled (which I only used to install snapd and upgrade related packages).

user@debian:~$ aptitude search "^mongodb"
p   mongodb                                          - object/document-oriented database (metapackage)
p   mongodb-clients                                  - object/document-oriented database (client apps)
p   mongodb-dev                                      - MongoDB C++ Driver (transitional package)
p   mongodb-server                                   - object/document-oriented database (server package)

Works without sudo, actually:
user@debian:~$ /snap/rocketchat-server/current/bin/mongo
MongoDB shell version: 3.2.7
connecting to: test
Welcome to the MongoDB shell.
...